packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
```

Comment:
I used an unordered list to mark up the tab links since that's my preferred way to organize the tabs and it's also what's shown in this article that was linked from the CRD: https://inclusive-components.design/tabbed-interfaces/

The linter is complaining though. (I would prefer to keep the structure as is but of course will change it if necessary for accessibility.)

Reply:
@KristinLBradley let's turn off those two linting rules specifically for this component only.

Reply:
@MelSumner  Thanks. How/where do I turn them off?

Reply:
Sorry I figured out how to turn them off. Thanks.